tests/test_helper/microovn.bash

```diff
@@ -44,6 +44,23 @@ function bootstrap_cluster() {
         echo "# Joining cluster with $container" >&3
         lxc_exec "$container" "microovn cluster join $token"
     done
+
+    # Make sure that services are up after the initial bootstrap
```

Comment:
The code base contains multiple ways of bootstrapping the cluster, does this perhaps belong in a separate function so that it can be used elsewhere as well?

Reply:
Code block extracted to function `wait_ovn_services` in `microovn.bash`